ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -21,6 +22,7 @@ module('Acceptance | sync | overview', function (hooks) {
   hooks.beforeEach(async function () {
     this.version = this.owner.lookup('service:version');
     this.version.features = ['Secrets Sync'];
+    this.permissions = this.owner.lookup('service:permissions');
```

Comment:
I think you may also want to set `this.version.type = "enterprise"' because otherwise the `showSecretsSync` method returns false before it gets to license or flag features, rendering the permissions redirect test check somewhat moot

Reply:
ah, great catch. will update now. thanks!